tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -117,10 +117,6 @@ def run_pipeline_test(self, speech_recognizer, examples):
                     "chunks": [{"text": ANY(str), "timestamp": (ANY(float), ANY(float))} for i in range(n)],
                 },
             )
-        else:
-            # Non CTC models cannot use return_timestamps
-            with self.assertRaisesRegex(ValueError, "^We cannot return_timestamps yet on non-ctc models !$"):
-                outputs = speech_recognizer(audio, return_timestamps="char")
```

Comment:
You're removing very interesting negative test. Just test for whisper. but don't remove good sanity checks :D

Reply:
Haha yeah you're right 😅